streams/shared/src/main/scala/scalaz/stream/ZStream.scala

```diff
@@ -185,6 +185,11 @@ trait ZStream[-R, +E, +A] extends Serializable { self =>
   final def foreach[R1 <: R, E1 >: E](f: A => ZIO[R1, E1, Unit]): ZIO[R1, E1, Unit] =
     foreachWhile(f.andThen(_.const(true)))
 
+  /**
+   * Ignores a stream failure and return `Unit`
+   */
+  final def ignore: ZIO[R, Nothing, Unit] = self.run(Sink.drain).either.unit
```

Comment:
I would not add this. I think we're missing the error combinators necessary to do this one right.

Reply:
Hey John I removed the function, do you think we can create the missing combinators to make *ZStream* `ignore` in this issue?